atomic_reactor/plugins/build_source_container.py

```diff
@@ -42,52 +33,20 @@ def run(self):
         # TODO fail when source dir is empty
 
         image_output_dir = tempfile.mkdtemp()
-        image = self.get_builder_image()
-        if not image:
-            raise RuntimeError(
-                'Cannot build source containers, builder image is not '
-                'specified in configuration')
-
-        image = ImageName.parse(image)
-
-        srpms_path = '/data/'
-        output_path = '/output/'
-
-        volume_bindings = {
-            source_data_dir: {
-                'bind': srpms_path,
-                'mode': 'ro,Z',
-            },
-            image_output_dir: {
-                'bind': output_path,
-                'mode': 'rw,Z',
-            }
-        }
-
-        pulled_img = self.tasker.pull_image(image)
-
-        command = '-d sourcedriver_rpm_dir -s {srpms_path} -o {output_path}'.format(
-            srpms_path=srpms_path,
-            output_path=output_path,
-        )
-        container_id = self.tasker.run(
-            pulled_img,
-            volume_bindings=volume_bindings,
-            command=command
-        )
-        status_code = self.tasker.wait(container_id)
-        output = self.tasker.logs(container_id, stream=False)
 
-        self.log.debug("Build log:\n%s", "\n".join(output))
+        cmd = ['bsi',
```

Comment:
How about this option?

`-d sourcedriver_rpm_dir`

Reply:
Added! I had missed this one.